protocol/app/app.go

```diff
@@ -603,7 +604,8 @@ func New(
 				daemonservertypes.MaximumAcceptableUpdateDelay(daemonFlags.Liquidation.LoopDelayMs),
 			)
 			go func() {
-				if err := liquidationclient.Start(
+				app.LiquidationsClient = liquidationclient.NewClient()
```

Comment:
should this live outside the `go func()` to be consistent w the price daemon [here](https://github.com/dydxprotocol/v4-chain/pull/738/files#diff-7148577e1c863205cc155a436a9c9b806e655073d37a5124da0af8b5e9370317R629)?

Also, as a follow up PR, can we create a similar method that starts the liquidation daemon like [this](https://github.com/dydxprotocol/v4-chain/blob/4bd3c34a0bf385e91bf5fb509115e19dcf9bd44a/protocol/daemons/pricefeed/client/client.go#L267)? We should log the liquidation daemon flags too

Reply:
See comment about `StartNewClient` below.